configs/rabbitmq/rabbitmq.conf

```diff
@@ -0,0 +1 @@
+load_definitions = /etc/rabbitmq/definitions.json
```

Comment:
can this be a relative path?

```
load_definitions =./definitions.json
```

to remove the need to match what is in the docker-compose volume mount?  just sorta assume its in the same directory as the `rabbitmq.conf`?

Reply:
I don't know if rabbitmq supports relative paths? I can try it.

In practice, I always mount all rabbitmq configs to /etc/rabbitmq since that's where rabbitmq looks for its config file. Just makes getting all the config in simpler.

Reply:
okay sounds good. leave it.